pkg/event/event.go

```diff
@@ -23,7 +23,7 @@ import (
 	"github.com/redhat-cne/sdk-go/pkg/types"
 )
 
-const APISchemaVersion = "1.0"
+const APISchemaVersion = "2.0"
```

Comment:
do not change this. O-RAN currently use 1.0 for data schema version. It is different from REST API version.